src/components/snapshots/useSnapshot.ts

```diff
@@ -63,42 +60,68 @@ const useSnapshot: SnapshotOperation = async ({
       throw `Snapshot was created with version: ${snapshotVersion}\n You can't install a snapshot created with a version newer than the current application version: ${config.version}`
     }
 
-    const snapshotObject = JSON.parse(snapshotRaw)
-
-    if (options.shouldReInitialise) {
-      await initialiseDatabase(options, snapshotFolder)
-    }
-
-    if (options.resetFiles) {
+    if (options.resetFiles || options.usePgDump) {
       execSync(`rm -rf ${FILES_FOLDER}/*`)
     }
 
-    // Prevent triggers from running while we insert data, but only for full re-init
-    if (options.shouldReInitialise) {
+    if (options.usePgDump) {
+      // The quick way, using pg_restore -- whole database only
+      console.log('Restoring database...')
+      execSync(
+        `pg_restore --clean --if-exists --dbname tmf_app_manager ${snapshotFolder}/database.dump`
```

Comment:
`--clean` drops everything in the database before restore
`--if-exists` means that it won't throw errors if the objects it's dropping (with `clean`) don't exists (i.e. if loading into an already clean database)